pkg/subctl/cmd/join.go

```diff
@@ -458,6 +462,18 @@ func populateSubmarinerSpec(subctlData *datafile.SubctlData, netconfig globalnet
 	if len(customDomains) > 0 {
 		submarinerSpec.CustomDomains = customDomains
 	}
+	if len(imageOverrideArr) > 0 {
+		imageOverrides = make(map[string]string)
+		for _, s := range imageOverrideArr {
+			key := strings.Split(s, "=")[0]
+			value := strings.Split(s, "=")[1]
+			imageOverrides[key] = value
+		}
+	} else {
+		imageOverrides = nil
+	}
+	submarinerSpec.ImageOverrides = imageOverrides
```

Comment:
You can even move this assignment inside the if branch: after L471

Reply:
Done